build_env.py

```diff
@@ -0,0 +1,12 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
```

Comment:
Thanks for the pointers. It seems that running ``nodeenv`` with ``-p`` recreates the node environment every time. And trying to do ``nodeenv {dest_dir}`` causes and error if the environment already exists.

I did some digging and found that one can suppress errors in tox by adding a ``-`` prefix to a command. So at least now we don't need the ``build_env.py`` script anymore.